### DISCUSSION TITLE
Chrome Version/Driver on Win22 Image

### DISCUSSION THREAD

**Discussion by jordanabakerafs** [actions/virtual-environments#7688] [Installed software]
Our teams have switched to version 114 of Chrome to mitigate vulnerabilities, however the runner software roster still lists 113.0.5672.93.

Is there a means to determine when the runner image may have it's chrome and chrome driver version updated to the latest version?

**Comment by vpolikarpov-akvelon** [comment]
Hey @jordanabakerafs. There are pull requests for both versions of Windows Runner ([2019](https://github.com/actions/runner-images/pull/7680) and [2022](https://github.com/actions/runner-images/pull/7679)) that include updates for browsers and web drivers. It means that updates are underway already. Unfortunately we can't provide the exact date when rollout completes.

**Reply by jordanabakerafs** [reply]
Thanks @vpolikarpov-akvelon !
